src/views/RecIM/components/Forms/ActivityForm/index.js

```diff
@@ -278,6 +278,20 @@ const ActivityForm = ({
     setNewInfo(currentInfo);
   };
 
+  useEffect(() => {
+    const listener = (event) => {
+      if (event.code === 'Enter' || event.code === 'NumpadEnter') {
```

Comment:
I'm going to comment on this line just so that we can see replies instead of scrolling for a year. I honestly think that it would be a great addition to the GordonDialogBox. However, I do think that we should handle some sort of _timeout_ to ensure that the confirmation window doesn't instantly get submitted if the User presses enter too quickly. This shouldn't affect other DialogBoxes as I'm pretty sure the standard usecase revolves around actually filling out a form.

Reply:
I would be concerned with implementing this globally since it is not necessarily a feature that we would want on all dialog boxes. Instead, I would use a `closeDialog()` function passed into the child component (in these cases, a form), then handle the key press in the child component.